index.js

```diff
@@ -27,7 +27,7 @@ function findFirstDFS(compilation, key) {
   }
 }
 
-WebpackNotifierPlugin.prototype.compileEndOptions = function (stats) {
+WebpackNotifierPlugin.prototype.compileEndOptions = function compileEndOptions(stats) {
```

Comment:
Are these changes important for this PR?

Reply:
No changes, just lint and dumb deps